app/models/ruby_version.rb

```diff
@@ -29,6 +30,18 @@ def prerelease?
     dev? || prerelease
   end
 
+  def git_ref
+    git_tag.present? ? git_tag : git_branch
```

Comment:
I'm a big fan of `presence`, so that you don't have to reference `git_tag` twice.

https://github.com/rails/rails/blob/18707ab17fa492eb25ad2e8f9818a320dc20b823/activesupport/lib/active_support/core_ext/object/blank.rb#L45-L47

```suggestion
    git_tag.presence || git_branch
```